stdlib/public/runtime/ImageInspection.h

```diff
@@ -106,6 +108,47 @@ void addImageAccessibleFunctionsBlockCallbackUnsafe(const void *baseAddress,
 
 int lookupSymbol(const void *address, SymbolInfo *info);
 
+#if defined(_WIN32)
+/// Configure the environment to allow calling into the Debug Help library.
+///
+/// \param body A function to invoke. This function attempts to first initialize
+///   the Debug Help library. The result of that operation is passed to this
+///   function.
+///
+/// On Windows, the Debug Help library (DbgHelp.lib) is not thread-safe. All
+/// calls into it from the Swift runtime and stdlib should route through this
+/// function.
+SWIFT_RUNTIME_STDLIB_SPI
+void _swift_withWin32DbgHelpLibrary(
+  const std::function<void(bool /*isInitialized*/)>& body);
+
+/// Configure the environment to allow calling into the Debug Help library.
+///
+/// \param body A function to invoke. This function attempts to first initialize
+///   the Debug Help library. The result of that operation is passed to this
+///   function.
+///
+/// \returns Whatever is returned from \a body.
+///
+/// On Windows, the Debug Help library (DbgHelp.lib) is not thread-safe. All
+/// calls into it from the Swift runtime and stdlib should route through this
+/// function.
+template <
```

Comment:
And this is where I'd put my `if constexpr`… _if I had one._ (Only available in C++17, must use `std::enable_if` in its absence.)